tests/framework/common_apps.go

```diff
@@ -72,6 +102,22 @@ func (td *OsmTestData) GetMutatingWebhook(mwhcName string) (*v1beta1.MutatingWeb
 	return mwhc, nil
 }
 
+// GetPodsForAppLabel returns the Pods matching a specific `appLabel`
+func (td *OsmTestData) GetPodsForAppLabel(ns string, labelSel metav1.LabelSelector) ([]corev1.Pod, error) {
```

Comment:
Since this is generic for any label selector, not just an `app` label:
```suggestion
func (td *OsmTestData) GetPodsForLabels(ns string, labelSel metav1.LabelSelector) ([]corev1.Pod, error) {
```

Reply:
will fix.